ui/qt-lib/src/Ariadne/UI/Qt/Face.hs

```diff
@@ -172,27 +173,29 @@ data UiWalletTreeSelection =
     , wtsPath :: TreePath
     }
 
+data UiCurrency = ADA | Lovelace
```

Comment:
Hmm, this code was written before we went currency-unaware.
I could make Glue render currencies to text and display that text in the UI, but the reason I did it like this is to be able to select correct currency icon (see https://github.com/serokell/ariadne/pull/253/files/efc09ed03a04358567b0529fb1c404bb961c0639..fcd24a59273e2ef9f65adcbedabe2cb228dfd501#diff-dd3e1b734b89591508b1abe0d9624534R172).

I don't think Glue should be the place to select icons in...

Reply:
> Hmm, this code was written before we went currency-unaware.

Well, I'd say that it was currency-unaware from the beginning, because we intentionally didn't want it to depend on `cardano-sl-*` packages.

But now I'm looking at it and realize that it's hard to predict what we'll want in the future and when, so I think let's leave it as is and maybe change something when we actually need that.